core/mondoo-okta-security.mql.yaml

```diff
@@ -1,379 +1,1985 @@
+        Enable this email notification so end users are informed about new sign-on activity, which can inform them if a different user has signed in to their account.
 
-        2. Run this query:
+        ### Security impact 
 
-          ```mql
-          okta.policies.signOn.where(_.rules.all(status == "ACTIVE")).all(_.rules.all(actions["signon"]["factorPromptMode"] == "ALWAYS"))
-          ```
-      remediation: "### Use the Okta Admin Console\n\n1. Log into the organization as an administrator.\n2. Select **Security > Global Session Policy** \n3. Add a policy or make a change to the default policy. \n4. Assign the policy to **Everyone**.\n5. Select **Add rule** and make sure **Multifactor Authentication (MFA)** is set to **Required**.\n6. Under **Users will be prompted for MFA**, select **At every sign-in**.\n7. Make the status of the rule is **Active**.\n"
-  - uid: mondoo-okta-security-okta-disable-persist-cookies
-    title: Don't persist session cookies across browser sessions
-    impact: 40
-    mql: |
-      okta.policies.signOn.where(_.rules.all(status == "ACTIVE")).all(_.rules.all(actions["signon"]["session"]["usePersistentCookie"] == false))
+        High
+
+        ### User impact 
+
+        Low
+
+        End users receive an email notification if they sign in from a new or unrecognized client. 
+
+        ### Known limitations
+        Currently, new sign-on notifications don't use Improved New Device Behavior Detection when sending email notifications for new sign-ins. Changes to `deviceToken` or browser cookies may not trigger a new sign-on email notification.
+    variants: 
+      - uid: mondoo-okta-security-sign-on-notifications-for-end-users-runtime
+      - uid: mondoo-okta-security-sign-on-notifications-for-end-users-terraform-hcl
+      - uid: mondoo-okta-security-sign-on-notifications-for-end-users-terraform-plan
+  - uid: mondoo-okta-security-sign-on-notifications-for-end-users-runtime 
+    title: Enable sign-on notifications for end users 
+    impact: 80
+    filters: asset.platform == "okta-org"
+    mql: okta.organization.securityNotificationEmails['sendEmailForNewDeviceEnabled'] == true
     docs:
-      desc: "Persisting session cookies across browser sessions means that when a user closes and reopens a browser while a session is still active, they don't need to re-authenticate. \n\nSession cookies (non-persistent cookies) are stored in memory and never written to the disk. Session cookies remain active as long as the browser remains active; once the browser closes, the cookies vanish.\n"
-      audit: |
-        __Use `cnspec run`__
+      remediation: 
+        - id: console 
+          desc: |
+            ### Enable sign-on notification emails for end users
+            1. In the Admin Console, go to **Security > General**.
+            2. Under **Security Notification Emails**, click **Edit**.
+            3. Set **New sign-on notification email** to **Enabled**.
+            4. Click **Save**.
```

Comment:
```suggestion
            4. Select **Save**.
```

Reply:
@tas50 these are not our steps. they are from Okta's docs. They are copy/pasted.